docker/Dockerfile.service-java-run-base

```diff
@@ -24,7 +24,10 @@ RUN hail-pip-install -r service-base-requirements.txt
 
 COPY hail/python/setup-hailtop.py /hailtop/setup.py
 COPY hail/python/hailtop /hailtop/hailtop/
+COPY /hail_version /hailtop/hailtop/hail_version
+COPY hail/python/MANIFEST.in /hailtop/MANIFEST.in
 RUN hail-pip-install /hailtop && rm -rf /hailtop
+RUN echo 'from hailtop import version; print(version());' | python3
```

Comment:
Could you remove these echos? I get failing quickly but I'm hesitant to start adding validation like this into the Dockerfiles, as it's not exactly clear what should or should not be validated and it adds unused layers, so could get pretty messy quickly.

Reply:
Yep, removed in 7f487ad